README.md

```diff
@@ -67,6 +67,36 @@ python3 ./server.py 8880 &
 
 Then open <http://localhost:8880> in your browser.
 
+## Backporting Manual Changes
+
+In the event that a backported update is necessary, manual deployment of the newly built HTML is handled via Git tags, which are version-numbered tags based on major Lean versions matching the RegEx pattern `^v4\.\d+\.\d+$`. Pushing a new tag triggers the necessary deployment, but to do so requires first deleting any existing tag of the same name. The following describes the steps and commands you may use to manage this process.
```

Comment:
```suggestion
The manual for a Lean version is automatically deployed when a Git tag that matches the version is pushed. For Lean version `4.X.Y`, the manual is deployed from the contents of GIt tag `v4.X.Y` (note the leading `v`). In the event that a backported update is necessary, the tag should be changed. Pushing a new tag triggers the necessary deployment, but to do so requires first deleting any existing tag of the same name, because Git doesn't  support modification of existing tags. The following describes the steps and commands you may use to manage this process.
```